core/dbt/deps/base.py

```diff
@@ -97,6 +99,34 @@ def get_installation_path(self, project, renderer):
     def get_subdirectory(self):
         return None
 
+    def _install(self, project, renderer):
+        metadata = self.fetch_metadata(project, renderer)
+
+        tar_name = "{}.{}.tar.gz".format(self.package, self.version)
+        tar_path = os.path.realpath(os.path.join(get_downloads_path(), tar_name))
```

Comment:
Minor detail:  We're trying to avoid using `os.path` for new commits to dbt-core (preferring [Pathlib](https://docs.python.org/3/library/pathlib.html) instead).  THis is due to the many odd caveats when running in non-posix environments.